connectors/src/connectors/confluence/temporal/activities.ts

```diff
@@ -256,7 +256,15 @@ export async function confluenceCheckAndUpsertPageActivity({
 
   localLogger.info("Upserting Confluence page.");
 
+  // There is a small delta between the page being listed and the page being imported.
+  // If the page has been deleted in the meantime, we should ignore it.
   const page = await client.getPageById(pageId);
+  if (!page) {
+    localLogger.info("Confluence page not found.");
```

Comment:
nit:
`Confluence page ${pageId} of workflow ${workflowId} not found, we are ignoring the error to avoid breaking existing assistants.`

Reply:
So actually this is how our logger works, it's being initialized with parameters that will always be available in the logs. They can then be used to aggregate and search. We **ALWAYS** use static messages so it's easier to search for them in DD afterward.

Initiliazed like this:
https://github.com/dust-tt/dust/blob/6dd17448eb3bfbf5cc683f2d7fde71fe5cf3d750/connectors/src/connectors/confluence/temporal/activities.ts#L239

Will generate logs like [this](https://app.datadoghq.eu/logs?query=%22Upserting%20Confluence%20page.%22%20&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&event=AgAAAZGX77fZ0aQEUQAAAAAAAAAYAAAAAEFaR1g3N25OQUFCa1NrN1dIeW80MEFBTgAAACQAAAAAMDE5MTk3ZWYtYjg0Zi00ZDBmLWIxYTItNTlhYWZjMmU3OWYy&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&from_ts=1724830060292&to_ts=1724830960292&live=true).